src/components/Leaderboard/LeaderboardRow.tsx

```diff
@@ -1,16 +1,50 @@
 import React from "react";
-import { LeaderboardUser } from "./Leaderboard";
+import {LeaderboardUser} from "./Leaderboard";
+import leaderboardL from "../../images/leaderboard-left.svg";
+import leaderboardLMe from "../../images/leaderboard-left-me.svg";
+import leaderboardR from "../../images/leaderboard-right.svg";
+import useSWR from "swr";
+import {fetcher} from "../../services/fetch";
+import {Loading} from "../Loading/Loading";
 
-export const LeaderboardRow = (props: { rank: number, player: LeaderboardUser }) =>
-  <div
-    className="px-4 grid grid-cols-6 h-[54px] gap-2 text-xs flex items-center text-white border-b border-[#060968]">
-    <div className="text-sm">{props.rank + 1}.</div>
-    <div className="col-span-3 flex flex-col">
-      <span>{props.player.firstName}</span>
-      <span>{props.player.lastName}</span>
-    </div>
-    <div className="text-right col-span-2">
-      <span className="text-base font-text text-grey-4 mr-2 -mt-2">{Math.round(props.player.score.time / 1000)}s</span>
-      <span className="text-txs text-white align-bottom">{props.player.score.count}</span>
+const currentUser = () => {
+  const { data } = useSWR(`/members/me`, fetcher);
+  return {
+    me: data,
+    isLoading: !data
+  };
+};
+
+const isItMe = (me : LeaderboardUser, player: LeaderboardUser) => {
+  return me.firstName === player.firstName && me.lastName === player.lastName && me.picture === player.picture
+}
+
+export const LeaderboardRow = (props: { rank: number, player: LeaderboardUser }) => {
+  const { me, isLoading } = currentUser()
```

Comment:
Nous ne pouvons pas faire ça ; à chaque ligne "Loading" est affiché (même rapidement) et il y a potentiellement un appel réseau (grâce à useSWR fait qu'une fois) il faudrait récupérer l'utilisateur plutôt dans la page et le passer en paramètre (isCurrentUser par exemple).

Reply:
Fixed in #7fca777